Specs/Sequence/English/IpAddressModel.json

```diff
@@ -328,6 +328,41 @@
     "NotSupportedByDesign": "python",
     "Results": []
   },
+  {
+    "Input": "wrong IPV6 address :",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": "I say:",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": "I said :",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": "lync:",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": "lync: :",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": ":as you say",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": ": as you say",
```

Comment:
Can you add a time case, like "the train arrives at 10:00"?

Reply:
added